dev/bots/run_fuchsia_tests.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+# Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
2020?

Reply:
our policy (for engine at least) is to have 2014 for all headers. not sure why.

Reply:
I copied this over from the other shell scripts in this directory. When they get updated someone will likely resolve this script as well.